appveyor.yml

```diff
@@ -34,12 +34,13 @@ before_deploy:
   - move /Y %APPVEYOR_BUILD_FOLDER%\packaging\windows\OpenRA.Setup.exe %APPVEYOR_BUILD_FOLDER%\OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
 
 artifacts:
-  - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
+  - name: Installer
+    path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe

```

Comment:
just to be safe, I'd turn those two lines around. That's the way it's always shown in the appveyor docs. Not sure if it makes a difference, but...